src/main/java/hudson/plugins/accurev/CheckForChanges.java

```diff
@@ -295,4 +192,42 @@ public static boolean checkStreamForChanges(AccurevServer server,
         }
         return isTransLatestThanBuild;
     }
+
+    private static boolean changesMatchFilter(TaskListener listener, Collection<String> serverPaths, List<String> filters) {
+        if (filters == null || filters.isEmpty()) {
+            // No filters, so always a match.
+            return true;
+        }
+
+        for (String filterPath : filters) {
+            // Paths are sanitized using java.nio.file.Paths (makes sure that the same path separators are used)
+            final Path fp = Paths.get(filterPath);
+            for (String serverPath : serverPaths) {
+                final Path sp = Paths.get(serverPath);
```

Comment:
I'm not sure which one is faster though.
I would not be surprised if a string replace is faster, because it does not have to do any interpretation of the path.

I have currently rewritten the code to make more use of the Commons lib and have removed the Path object.
Reviewing my changes now, then I'll do a quick test and update this PR.